policies/approval.yml

```diff
@@ -110,7 +110,6 @@ approval_rules:
           - "product-os/policies"
           - "product-os/admins"
           - ".*/.github"
-          - "balena-io/environment-production"
```

Comment:
will this also auto-merge anything Renovate bot auto-approves in priduction?

Reply:
no, policy-bot does not do any merging

it will be down to renovates automerge rules [here](https://github.com/balena-io/environment-production/blob/master/.github/renovate.json) which disables automerge for everything then manually enables for certain packages.